scripts/cmscp.py

```diff
@@ -1254,7 +1257,7 @@ def update_exit_info(exit_info, exit_code, exit_msg = None, force = False):
     ## b) adding a 'failed' subdirectory in case cmsRun failed.
     if G_JOB_WRAPPER_EXIT_CODE != 0:
         dest_temp_dir = os.path.join(dest_temp_dir, 'failed')
-
+    dest_final_dir = os.path.join(dest_final_dir, counter)
```

Comment:
And please modify the comment above to `## Modify the stageout temporary and final directories by:`